drivers/census_tracking/app/models/census_tracking/worksheet.rb

```diff
@@ -0,0 +1,90 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module CensusTracking
+  class Worksheet
+    include ArelHelper
+
+    def initialize(filter)
+      @filter = filter
+    end
+
+    def projects
+      projects = GrdaWarehouse::Hud::Project.
+        preload(:organization).
+        viewable_by(@filter.user).
+        where(id: @filter.effective_project_ids)
+
+      projects.map { |project| [HUD.project_type(project.compute_project_type), project.organization.name, project.safe_project_name, project.id] }.
+        sort_by { |project| [project[0], project[1], project[2]] }
+    end
+
+    def clients_by_project(project_id, query)
+      scope = service_scope.in_project(project_id)
+      clients_by_population(query, scope: scope)
+    end
+
+    def clients_by_project_type(project_type, query)
+      scope = service_scope.in_project_type(HUD.project_type(project_type, true))
+      clients_by_population(query, scope: scope)
+    end
+
+    def clients_by_population(query, scope: service_scope)
+      scope.where(query)
+    end
+
+    def populations  # rubocop:disable Metrics/AbcSize
+      @populations ||= {
+        'Unaccompanied Males Under Age 18' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].lt(18)).and(c_t[:Gender].eq(1)),
+        'Unaccompanied Trans Males Under Age 18' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].lt(18)).and(c_t[:Gender].eq(3)),
+        'Unaccompanied Females Under Age 18' => she_t[:presented_as_individual].eq(true).and(shs_t[:age].lt(18)).and(c_t[:Gender].eq(0)),
```

Comment:
These ages probably aren't correct.  We'll want age on the chosen date, not age at entry

Reply:
See filter_scopes.rb#filter_for_age for the logic

Reply:
Oh, never mind, you are using shs_t, not she_t.